web_searchbar_full_width/static/src/css/web_searchbar_full_width.css

```diff
@@ -1,20 +1,4 @@
-.openerp .oe_application .oe_breadcrumb_item:not(:last-child)
-{
-    max-width: inherit;
-}
```

Comment:
this made the bread crumb full width, you probably need something similar as below for `div.oe-cp-title` in v9. As it is now, the breadcrumb is only half the width of the search bar